app-admin/kubelet-wrapper/files/kubelet-wrapper

```diff
@@ -4,36 +4,72 @@
 # Make sure to set KUBELET_VERSION to an image tag published here:
 # https://quay.io/repository/coreos/hyperkube?tab=tags Alternatively,
 # override $KUBELET_ACI to a custom location.
+#
 
 set -e
 
-if [ -z "${KUBELET_VERSION}" ]; then
-    echo "ERROR: must set KUBELET_VERSION"
-    exit 1
+function require_ev_all() {
+	for rev in $@ ; do
+		if [[ -z ${!rev} ]]; then
+			echo ${rev} is not set
+			exit 1
+		fi
+	done
+}
+
+function require_ev_one() {
+	for rev in $@ ; do
+		if [[ ! -z ${!rev} ]]; then
+			return
+		fi
+	done
+	echo One of $@ must be set
+	exit 1
+}
+
+if [[ ! -z ${KUBELET_VERSION} ]]; then
+	echo KUBELET_VERSION environment variable is deprecated, please use KUBELET_IMG_TAG instead
+fi
+
+if [[ ! -z ${KUBELET_ACI} ]]; then
+	echo KUBELET_ACI environment variable is deprecated, please use the KUBELET_IMG instead
 fi
 
-KUBELET_ACI="${KUBELET_ACI:-quay.io/coreos/hyperkube}"
+KUBELET_IMG_TAG="${KUBELET_IMG_TAG:-${KUBELET_VERSION}}"
+KUBELET_IMG="${KUBELET_ACI}"
+
+require_ev_one KUBELET_IMG KUBELET_IMG_TAG
+
+if [[ ! -z ${KUBELET_IMG_TAG} ]]; then
+	KUBELET_IMG_URL=${KUBELET_IMG_URL:-"quay.io/coreos/hypberkube"}
+	KUBELET_IMG="${KUBELET_IMG_URL}:${KUBELET_IMG_TAG}"
+	RKT_RUN_ARGS="${RKT_RUN_ARGS} --trust-keys-from-https"
```

Comment:
It's assumed that when they use the tag, they use our default image URL which needs that argument. If the user provides KUBELET_IMG_URL directly, this flag isn't necessarily needed.

What this should probably check instead is whether or not KUBELET_IMG_URL is set.